drivers/ethernet/eth_w5500.c

```diff
@@ -347,9 +347,10 @@ static int w5500_set_config(const struct device *dev,
 	if (IS_ENABLED(CONFIG_NET_PROMISCUOUS_MODE) &&
 	    type == ETHERNET_CONFIG_TYPE_PROMISC_MODE) {
 		if (config->promisc_mode) {
-			if (!(mode & BIT(mr)))
+			if (!(mode & BIT(mr))) {
 				return -EALREADY;
 			}
+			}
```

Comment:
formatting here seems off

Reply:
oh man, the fun of applying clang-format only on changed files assuming the original was ok